.clang-format

```diff
@@ -4,7 +4,7 @@ BasedOnStyle:  Google
 
 # Setting ColumnLimit to 0 so developer choices about where to break lines are maintained.
 # Developers are responsible for adhering to the 120 character maximum.
-ColumnLimit:     0
+ColumnLimit:     120
```

Comment:
Is this the new limit that we want to adopt? If so, the comment should reflect the intention.

Reply:
I will revert this part for now. It was helpful to write lines that are not long.

Reply:
However, I would recommend 120 to align with ORT code base.

Reply:
>  It was helpful to write lines that are not long.

I much appreciate visual indicators for line length, which is why I always enabled VS Code's vertical rulers option: `"rulers": [80, 120]`.

![image](https://github.com/user-attachments/assets/548d4812-2356-4638-9694-8e8438029518)

> \# Setting ColumnLimit to 0 so developer choices about where to break lines are maintained.

But given guides, I lean toward trusting developers to make informed choices and consciously writing more readable code than formatting bulldozers would, which often unnecessarily fragment lines into ragged spaghetti because a line is a few characters beyond the norm :(.